src/api/ledger/accountinfo.js

```diff
@@ -6,7 +6,43 @@ const removeUndefined = require('./parse/utils').removeUndefined;
 const validate = utils.common.validate;
 const composeAsync = utils.common.composeAsync;
 
-function formatAccountInfo(response) {
+type AccountData = {
+  Sequence: number,
+  Account: string,
+  Balance: string,
+  Flags: number,
+  LedgerEntryType: string,
+  OwnerCount: number,
+  PreviousTxnID: string,
+  AccountTxnID: string,
+  PreviousTxnLgrSeq: number,
+  index: string
+}
+
+type AccountDataResponse = {
+  account_data: AccountData,
+  ledger_current_index: number,
```

Comment:
okay, then I will make `ledger_current_index` optional, and add optional `ledger_hash` field

and, `getAccountInfoAsync` doesn't allow to specify `validated', or ledger hash, as ledger index in request - maybe we should change `ledgerVersion` schema?